proprietary/tokens/src/components/ams/footer.tokens.json

```diff
@@ -1,7 +1,29 @@
 {
   "ams": {
     "footer": {
-      "top": {
+      "menu": {
+        "column-gap": { "value": "{ams.space.lg}" },
+        "row-gap": { "value": "{ams.space.xs}" },
+        "link": {
```

Comment:
We sort groups after individual style declarations, so `link` should come after `padding-block` and `padding-inline`.